packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -82,18 +84,22 @@ export class AtomicRatingFacet
    */
   @Prop() public field!: string;
   /**
-   * The number of stars to request for this facet.
+   * The number of intervals to split the index for this facet.
    */
-  @Prop() public numberOfStars = 5;
+  @Prop() public numberOfIntervals = 5;
   /**
-   * The maximum value of the field. This value is used to normalize the field values with the number of stars.
+   * The maximum value of the field. This value is also used as the number of icons to be displayed.
```

Comment:
With @olamothe comment the other day (IIRC?), this maxValueInIndex should be removed and only the numberOfIntervals can be used for the scale & number of icons, this will simplify the code even more

Reply:
With this morning discussions, I guess we can keep the 2 props in order to be more extensible.
An addition, if the the maxValueInIndex is not defined, can it default to numberOfIntervals instead of 5? So users won't always have to configure those 2 props

Reply:
Sounds good 👍